rasa/core/test.py

```diff
@@ -542,7 +542,7 @@ def _log_stories(
             f.write("\n\n")
 
 
-async def test(
+async def evaluate_stories(
```

Comment:
Having a function `test` in the module `test` was definitely a bad decision back then 🙈 However, we can't just change it as it's somewhat part of the public API of Rasa Open Source. Do we need that for the tests to pass on Windows? Might also make sense to create an issue to handle this separately